File: ProgramingBasics/FirstStepsInCoding/04.TriangleOf55Stars/src/TriangleOfStars.java
public class TriangleOfStars {
    public static void main(String[] args) {
        for (int i=1; i<=10; i++) {
            for (int j=0; j<i; j++) {
                System.out.print("*");
            }
            System.out.println("");
        }
    }
}

File: ProgramingBasics/FirstStepsInCoding/04.TriangleOf55Stars/04.TriangleOf55Stars/TriangleOfStars.cs
using System;

namespace _04.TriangleOf55Stars
{
    public class TriangleOfStars
    {
        public static void Main(string[] args)
        {
            for (int i = 1; i <= 10; i++)
            {
                for (int j = 0; j < i; j++)
                    Console.WriteLine("*");

                Console.WriteLine("");
            }
        }
    }
}
